### PR TITLE
[wpmlcore-5204] Moved code for saving translated post body from Gitla…

### PR DESCRIPTION
**Pull request by adrianosferreira**
…b to Github

### CODE REVIEW

**Comment by adrianosferreira** [comment]
@OnTheGoSystems/wpml please review this. When moving things from ST to Github somehow this was not moved.

File: src/st/class-page-builder-integration.php [hunk 1]
```diff
@@ -5,6 +5,8 @@
  */
 class WPML_Page_Builders_Integration {
 
+	const UPDATE_TRANSLATED_POST_PRIORITY = 10;
```
Comment (sciamannikoo) [inline src/st/class-page-builder-integration.php]: @adrianosferreira shouldn't the constant reflect the name of the action isntead of the callback?

**Comment by sciamannikoo** [comment]
Except for the constant's name, there isn't much to see here, which is good :)

File: src/st/class-page-builder-integration.php (hunk 1 above)
Comment (adrianosferreira) [reply src/st/class-page-builder-integration.php]: @sciamannikoo makes sense, fixed.

**Pull request merged**